testkit/nodekit/node.go

```diff
@@ -15,6 +15,12 @@ import (
 	"go.uber.org/fx"
 )
 
+var (
+	BridgeNodeType = node.Bridge
+	FullNodeType   = node.Full
+	LightNodeType  = node.Light
+)
+
```

Comment:
Can you elaborate why this is needed? 
I propose to revert this back to vanilla node's Types

In the future Super Light Nodes and Partial nodes will require us adding them here